packages/components/src/alignment-matrix-control/icon.tsx

```diff
@@ -42,7 +44,6 @@ export default function AlignmentMatrixControlIcon( {
 			className={ classes }
 			disablePointerEvents={ disablePointerEvents }
 			role="presentation"
-			size={ size }
```

Comment:
while `size` is used earlier in this file, it doesn't actually get used by `Root`. Including it here also sets off TypeScript, because `Root` renders as a `div` which doesn't expect a `size` property.

Leaving it off of this prop list eliminates the error and shouldn't have any negative consequences.